mvnw.cmd

```diff
@@ -0,0 +1,205 @@
+@REM ----------------------------------------------------------------------------

```

Comment:
We don't support Windows and we know that Opencast won't build on Windows. We probably don't need this file, do we?

Reply:
I will remove the file, I almost removed it from the staging area.